### PR TITLE
MDJ Table Tooltip

### PR DESCRIPTION
**Pull request by marioaks**
Adds correct tooltip to the MDJ Table.
Adds some very minor style updates.

### CODE REVIEW

File: src/css/visualization.css [hunk 1]
```diff
@@ -170,45 +171,12 @@
...
-  display: flex;
-  align-items: center;
-}
-
-.dist-tooltip .tooltip-cell {
-  padding: 0.5em 0;
-}
+/* .dist-bar-container:hover {
+  border: 1px #fff solid;
+} */
```
Comment (alexwgraves) [inline src/css/visualization.css]: Remove commented-out stuff

File: src/css/visualization.css [hunk 2]
```diff
@@ -221,15 +189,16 @@
...
 }
 
 .nonmonetary-bar {
   background-color: rgba(48, 48, 48, 1);
 }
 
 .ror-bar {
-  background-color: rgba(176, 140, 240, 1);
+  background-color: rgba(193, 78, 248, 1);
+
```
Comment (alexwgraves) [inline src/css/visualization.css]: Remove extra return

**Review by alexwgraves — CHANGES_REQUESTED** [review]
Small styles changes

**Comment by alexwgraves** [comment]
In the future please wait for an approving review before merging!

**Comment by marioaks** [comment]
Oh I thought it was approved, my bad. Made those small style changes

**Pull request merged**